### PR TITLE
Set display name for spec CRD

### LINKED ISSUE #1007: CRD descriptions for both ServiceBinding resources should be updated
### What is the environment (Minikube, Openshift)?

OpenShift

### What is the SBO version used?

0.9.1

### What are the steps to reproduce this issue?
  1. OpenShift Console
  2. Install ServiceBinding operator
  3. The installation details shows two CRDs with a similar description

### What is the expected behaviour?

The description should be updated to distinguish the CRDs

### What is the actual behaviour?

Both descriptions are similar.

### Additional Information (Screenshots, etc)

Screen Shot 2021-08-09 at 1 26 48 PM

### PR DESCRIPTION
**Pull request by baijum**
Signed-off-by: Baiju Muthukadan <baiju.m.mail@gmail.com>

### Motivation

In the OpenShift console, display name helps the user to identify the API name.

### Changes

Set displaName attribute.

Fixes #1007

### CODE REVIEW

File: config/manifests/bases/service-binding-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ spec:
       name: servicebindings.binding.operators.coreos.com
       version: v1alpha1
     - description: Service Binding for Kubernetes. Describes the connection between a Provisioned Service and an Application Projection. Implements Service Binding Specifcation for Kubernetes (https://github.com/k8s-service-bindings/spec).
-      displayName: Service Binding
+      displayName: Service Binding (Spec API)
```
Comment (baijum) [inline config/manifests/bases/service-binding-operator.clusterserviceversion.yaml:24]:
```suggestion
      displayName: Service Binding - Spec API (Preview)
```

File: config/manifests/bases/service-binding-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ spec:
       name: servicebindings.binding.operators.coreos.com
       version: v1alpha1
     - description: Service Binding for Kubernetes. Describes the connection between a Provisioned Service and an Application Projection. Implements Service Binding Specifcation for Kubernetes (https://github.com/k8s-service-bindings/spec).
```
Comment (slemeur) [inline config/manifests/bases/service-binding-operator.clusterserviceversion.yaml:23]: (Preview) Service Binding Implementing specification for Kubernetes (https://github.com/k8s-service-bindings/spec). Describes the connection between a Provisioned Service and an Application Projection.

**Review by pedjak — APPROVED** [review]

**Comment by pedjak** [comment]
With the PR change, OpenShift console renders the following:

image

/cc @slemeur are we good with that?

**Comment by slemeur** [comment]
LGTM

**Comment by pedjak** [comment]
@baijum can you please squash the commits so that we can merge this PR?

**Comment by baijum** [comment]
> @baijum can you please squash the commits so that we can merge this PR?

Done.

**Pull request merged**